components/KiboAppHeader.vue

```diff
@@ -138,8 +143,13 @@
               <SfIcon class="sf-header__icon">
                 <font-awesome-icon :icon="[accountIcon, 'user-circle']" class="fa-icon" />
               </SfIcon>
-              <div class="kibo-header__icon">
-                <span class="kibo-header__icon-name"> My Account</span>
+              <div class="kibo-header__icon" v-if="!isAuthenticated">
+                <span class="kibo-header__icon-title">{{ $t("My Account") }}</span>
+                <span class="kibo-header__icon-name">{{ $t("Log In") }}</span>
+              </div>
+              <div class="kibo-header__icon" v-if="isAuthenticated">
+                <span class="kibo-header__icon-title">{{ `Hi, ${user.firstName}` }}</span>
```

Comment:
I think(not sure) for 'Hi' also localization is required.

Reply:
Resolved